### DISCUSSION TITLE
Coordenadas cielo vacío

### DISCUSSION THREAD

**Discussion by QuicoHernandez** [Stellarium/stellarium#3683] [Ideas]
Hola, referente a la orientación del encuadre de cometas.  Hay situaciones que cuando tenemos el cometa bien encuadrado, necesitamos pinchar en el centro del encuadre para que coja las coordenadas y llamarlas desde NINA pero en ese centro no hay objetos, está el cielo limpio y no hay coordenadas porque Stellarium no ve nada en ese sitio, obligando a escoger una estrella lo más próxima a ese punto del centro del encuadre y ya no es el encuadre que quiero. Mi pregunta es,  se puede hacer que Stellarium coja las coordenadas de un punto vacío sin objetos.

**Comment by gzotti** [comment]
Set a marker.
